cub/cub/device/dispatch/tuning/tuning_select_if.cuh

```diff
@@ -830,7 +1544,7 @@ struct policy_hub
       decltype(select_agent_policy<sm80_tuning<InputT,
                                                is_flagged<FlagT>(),
                                                are_rejects_kept<KeepRejects>(),
-                                               classify_offset_size<OffsetT>(),
+                                               offset_size::_4, // before SM100, we only tuned for int32
```

Comment:
But I hardcode the offset type size to 4 for Policy 800 and 900, so partition will use the int32 tunings.

Reply:
Looks good! Thank you.